pvlib/spa.py

```diff
@@ -21,23 +21,15 @@ def nocompile(*args, **kwargs):
 
 if os.getenv('PVLIB_USE_NUMBA', '0') != '0':
     try:
-        from numba import jit, __version__
+        from numba import jit
     except ImportError:
         warnings.warn('Could not import numba, falling back to numpy ' +
                       'calculation')
         jcompile = nocompile
         USE_NUMBA = False
     else:
-        major, minor = __version__.split('.')[:2]
-        if int(major + minor) >= 17:
-            # need at least numba >= 0.17.0
-            jcompile = jit
-            USE_NUMBA = True
-        else:
-            warnings.warn('Numba version must be >= 0.17.0, falling back to ' +
-                          'numpy')
-            jcompile = nocompile
-            USE_NUMBA = False
+        jcompile = jit
+        USE_NUMBA = True
 else:
     jcompile = nocompile
     USE_NUMBA = False
```

Comment:
Is there a reason for inconsistent usage of upper vs. lower case, e.g., `USE_NUMBA` vs. `has_numba`?

Reply:
The reason I see is is that `USE_NUMBA` is a module-level variable used inside a function far away from where it is defined while `has_numba` is used once in the same scope immediately after its definition.  But of course it's just style :)